src/transformers/models/bartpho/__init__.py

```diff
@@ -0,0 +1,35 @@
+# flake8: noqa
+# There's no way to ignore "F401 '...' imported but unused" warnings in this
+# module, but to preserve other warnings. So, don't check this module at all.
+
+# Copyright 2020 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import TYPE_CHECKING
+
+from ...file_utils import _LazyModule
+
+
+_import_structure = {
+    "tokenization_bartpho": ["BartphoTokenizer"],
+}
+
+
+if TYPE_CHECKING:
+    from .tokenization_bartpho import BartphoTokenizer
+
+else:
+    import sys
```

Comment:
```suggestion
from ...file_utils import _LazyModule, is_sentencepiece_available


_import_structure = {}

if is_sentencepiece_available():
    _import_structure["tokenization_bartpho"] = ["BartphoTokenizer"]

if TYPE_CHECKING:
    if is_sentencepiece_available():
        from .tokenization_bartpho import BartphoTokenizer

else:
    import sys
```

This tokenizer depends on `sentencepiece` so it should be imported only if `sentencepiece` is installed.

Reply:
Thanks. I will revise and commit.